Directory.Build.props

```diff
@@ -1,5 +1,6 @@
 <Project>
   <PropertyGroup>
-    <BatteriesVersion>100.5.0</BatteriesVersion>
+    <BatteriesVersion>154.1.0</BatteriesVersion>
```

Comment:
so this version number is _special_. It is supposed to be `{rust regex version}.{major}.{build}` or something like that. 

I'd see about adding a property `<CiBuild>` and setting that in the pipeline.

if we've moved to track rust regex 1.11.1 we should update this to be something like `1111.0.$(CiBuild)`.

And then when we pack do `/P:CiBuild=$SOME_FACNY_ENV_VAR`